firstbyte/urls.py

```diff
@@ -40,5 +40,5 @@
     path('checkin-2/', teacher_check_in_2_redirect),
     path('post-survey/', teacher_post_survey),
     path('', include('curriculum.urls')),
-    path('', include('events.urls')),
+    path('events/', team, name='team')
```

Comment:
Leave out `name='team'` here since that should be reserved for the `team/` endpoint. The name here is only used to retrieve the url from a template: aka `{% url 'team' %}` resolves to `team/`.